tests/integration/href-to-test.js

```diff
@@ -14,7 +14,7 @@ test(`#isNotLinkComponent should be true if the event target is not an instance
   this.render(hbs`{{not-a-link class='not-a-link'}}`);
 
   let event = leftClickEvent();
-  event.currentTarget = this.$('.not-a-link')[0];
+  event.target = this.$('.not-a-link')[0];
```

Comment:
No, it won't. But the goal of this PR was to **at least make the addon work in apps without jQuery**. 

Removing jQuery from the addon itself would require a bit more of work because it relies a lot in jQuery's `:contains` pseudo selector, but should be a reasonably mechanical task.